src/util-napatech.c

```diff
@@ -643,6 +644,7 @@ static void *NapatechStatsLoop(void *arg)
     }
 
     TmThreadsSetFlag(tv, THV_INIT_DONE);
```

Comment:
I think that this call can be removed but you'd have to test it. See `source-af-packet` (and other packet source modules for examples).